include/globals.h

```diff
@@ -1292,6 +1293,10 @@ extern RemoteDebug Debug;           // Let everyone in the project know about it
 #define ENABLE_OTA              1   // Listen for over the air update to the flash
 #endif
 
+#ifndef ENABLE_ESPNOW
+#define ENABLE_ESPNOW           0   // Listen for ESPNOW packets
```

Comment:
I'd suggest changing the comment to indicate the default is not listening to ESPNOW packets:

```suggestion
#define ENABLE_ESPNOW           0   // Don't listen for ESPNOW packets
```